internal/vendors/vendor.go

```diff
@@ -18,3 +21,11 @@ type Vendor struct {
 	ModifiedBy    int       `json:"modified_by"`
 	Date          time.Time `json:"dt"`
 }
+
+type GetAllVendorSpec struct {
+	Location string `json:"location"`
+	Product  string `json:"product"`
+	OrderBy  string `json:"order_by"`
+
+	database.PaginationSpec
+}
```

Comment:
I think we should append the ```OrderBy``` as paginationSpec because we want our page to show result in a consistent and predictable order. The ```order``` is already inside pagiantionSpec too